envs/default.yaml

```diff
@@ -1,22 +1,23 @@
 name: default
 channels:
-  - conda-forge
+    - conda-forge
 dependencies:
-  - python=3.6
-  - numpy=1.14.6
-  - pandas=0.23.2
-  - gdal=2.4
-  - pytest=3.2.3
-  - pytest-html=1.20.0
-  - ipython
-  - fiona=1.8.4
-  - rasterio=1.0.25
-  - rasterstats=0.13.0
-  - geopandas=0.8.2
-  - xarray=0.12.1
-  - netcdf4=1.5.1.2
-  - pyyaml=5.1
-  - pycountry=20.7.3
-  - pip=19.1.1
-  - pip:
-      - pytest-pythonpath # adds the root path "./" to PYTHONPATH
+    - python=3.6
+    - numpy=1.14.6
+    - pandas=0.23.2
+    - gdal=2.4
+    - pytest=3.2.3
+    - pytest-html=1.20.0
+    - ipython
+    - fiona=1.8.4
+    - rasterio=1.0.25
+    - rasterstats=0.13.0
+    - geopandas=0.8.2
+    - xarray=0.12.1
+    - netcdf4=1.5.1.2
+    - pyyaml=5.1
+    - pycountry=20.7.3
+    - pip=19.1.1
+    - pip:
+        - pytest-pythonpath # adds the root path "./" to PYTHONPATH
```

Comment:
Is this still necessary?

Reply:
probably not, but will need to check. Changing the default.yaml file is the slowest thing to test, since replacing the conda environment is slow as hell, so I try to avoid it where possible ;)